demo/src/main/java/com/google/android/fhir/demo/care/CarePlanManager.kt

```diff
@@ -273,16 +273,18 @@ class CarePlanManager(
 
   private suspend fun updateCarePlanStatus(
     carePlan: CarePlan,
-    requestResource: Resource,
-    carePlanActivityStatus: CarePlanActivityStatus
+    requestedActivityResource: Resource,
+    carePlanActivityStatus: CarePlanActivityStatus,
+    outcomeActivityReferences: List<Reference>
```

Comment:
Maybe we should use `outcomeReferences` instead? Using `Activity` might be a bit confusing because of `ActivityDefinition` resources.